src/script.js

```diff
@@ -3,25 +3,46 @@ import googleAnalytics from './analytics'
 import syncLibrary from './sync-library'
 
 // Replace layerStyles and textLayerStyles in the document with selected theme library
-export default function() {
-  const libraries = sketch.getLibraries().filter(l => l.valid && l.enabled)
-  const [, selectionIndex, ok] = sketch.UI.getSelectionFromUser(
-    "Select a theme library to replace magically document's layer styles and symbols 🎉",
-    libraries.map(l => l.name)
-  )
-  googleAnalytics('Open Camilo', 'Alert', 'UI')
-
-  // Depending selected control, current document will sync with predefined brand
-  if (ok) {
-    const doc = sketch.getSelectedDocument()
-    const chosenLibrary = libraries[selectionIndex]
-
-    googleAnalytics('Camilo replacement with', chosenLibrary.name, 'Library')
-
-    syncLibrary(doc, chosenLibrary)
+export function getOptionSelected(libraries) {
+  
+  const options = [];
+  var optionSelected = NSComboBox.alloc().initWithFrame(NSMakeRect(0, 0, 200, 25));
+  
+  libraries.forEach(function (lib) {
+    options.push(lib.name);
+  });
+  
+  optionSelected.i18nObjectValues = options;
+  optionSelected.setEditable(false);
+  optionSelected.addItemsWithObjectValues(options);
+  optionSelected.selectItemAtIndex(0);
+  return optionSelected;
+}
 
-    sketch.UI.message(
-      `🎉 🎈 🙌🏼  Applied theme from ${chosenLibrary.name}  🙌🏼 🎉 🎈`
-    )
+export default function(context) {
+  var PLUGIN_LOGO = 'icon.png';
+  const libraries = sketch.getLibraries().filter(l => l.valid && l.enabled)
+  var alert = COSAlertWindow.new();
+  alert.setMessageText('Camilo');
+  alert.setInformativeText("Select a theme library to replace magically document's layer styles and symbols 🎉")
+  alert.setIcon(NSImage.alloc().initByReferencingFile(context.plugin.urlForResourceNamed(PLUGIN_LOGO).path()));
+  alert.addAccessoryView(getOptionSelected(libraries));
+  alert.addButtonWithTitle('Sync');
+  alert.addButtonWithTitle('Cancel');
+  googleAnalytics(context, "Open Camilo", "Alert", "UI");
+  
+  // Depending selected control, current document will sync with predefined brand 
+  if (alert.runModal() == NSAlertFirstButtonReturn) {
+    var chosenLibrary = alert.viewAtIndex(0).stringValue();
+    libraries.forEach(function (lib) {
+      if (lib.name == chosenLibrary) {
```

Comment:
that'd be better written as
```js
const chosenLibraryName = String(alert.viewAtIndex(0).stringValue())
const lib = libraries.find(l => l.name === chosenLibraryName)
```

Reply:
Nice, I will change, not familiar yet with arrow functions.